Example/android/build.gradle

```diff
@@ -2,18 +2,18 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "27.0.3"
+        buildToolsVersion = "28.0.3"
         minSdkVersion = 16
         compileSdkVersion = 28
-        targetSdkVersion = 26
-        supportLibVersion = "27.1.1"
+        targetSdkVersion = 28
+        supportLibVersion = "1.0.2"
```

Comment:
There is no use for this in AndroidX. _Almost_ every AndroidX library has different versioning not like in Support Library where everything was chained together.
```suggestion

```